src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/RcaConfTest.java

```diff
@@ -104,4 +112,13 @@ public void testReadDeciderConfig() {
     Assert.assertFalse(workLoadTypeConfig.preferSearch());
     Assert.assertTrue(workLoadTypeConfig.preferIngest());
   }
+
+  @Test
+  public void testReadActionConfig() {
+    Map<String, Object> actionConfig = rcaConf.getActionConfigSettings();
+    NestedConfig cacheSettingsConfig = new NestedConfig("cache-settings", actionConfig);
+    Config<Integer> coolOffPeriodInSeconds = new Config<Integer>("cool-off-period-in-seconds", cacheSettingsConfig.getValue(),
```

Comment:
we can remove the cool off period config, now that DM doesn't handle it

Reply:
+1